### DISCUSSION TITLE
Getting error when trying to run "forge script script/DeployFundMe.s.sol". Check screenshot below

### DISCUSSION THREAD

**Discussion by HonourJAH** [foundry-rs/foundry#8785] [General]
![Screenshot from 2024-09-02 06-10-56](https://github.com/user-attachments/assets/600dab4e-4737-406d-aecd-cbb4ff6f884a)

**Comment by urosognjenovic** [comment]
Delete the ```cache``` folder and try to run the script again.

**Reply by HonourJAH** [reply]
I have fixed it 
Did exactly that earlier

Thank you so much
